docs/devices/TS0222.md

```diff
@@ -64,3 +64,24 @@ It's not possible to read (`/get`) or write (`/set`) this value.
 The minimal value is `0` and the maximum value is `255`.
 The unit of this value is `lqi`.
 
+## Issues
+
+### Not supported in HomeAssistant Zigbee2MQTT Addon
+
+- Add new file `TS0222.js` in `/config/zigbee2mqtt` with following content
+
+```const {battery, illuminance, linkQuality} = require('zigbee-herdsman-converters/lib/modernExtend');
+
+const definition = {
+    zigbeeModel: ['TS0222'],
+    model: 'TS0222',
+    vendor: '_TZ3000_8uxxzz4b',
```

Comment:
This device is supported in the latest z2m version: https://github.com/Koenkk/zigbee-herdsman-converters/blob/d42ffbd235102c7f9a6c8f4291a09f8cad38a681/src/devices/tuya.ts#L6849